taskRequests/details/style.css

```diff
@@ -0,0 +1,250 @@
+:root {
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.hidden {
+  display: none !important;
```

Comment:
May I know the use case for using `!important` here ?

Reply:
+ we should avoid `important` and rather try to increase specificity.

Reply:
hidden class is added via javascript, important flag here ensures that `hidden` is preferred over any other display property. The feature works without the important flag but this is just an extra check

Reply:
I think you should increase specificity rather than using `important`.

Reply:
If it's working without important, don't see the necessity for doing this.

Reply:
I think you should remove the importan from here...

Reply:
addressed this comment, please check